scripts/v2-migration/products/input/V2 Onchain Product Info - Products.csv

```diff
@@ -0,0 +1,84 @@
+Product Id,Product Address ,Name,Product Type,Yield Token Address,Cover Assets,Initial Price Ratio,Capacity Reduction Ratio,Is Deprecated,Use Fixed Price,Allowed Pools,IPFS Hash,IPFS data

```

Comment:
pls only include the info you need in this file 0 i.e. the exclusions. careful that the current spreadsheet contains multiple exclusion statements - you'll have to make each statement an item in the array

Reply:
I understand, but it's pretty much impossible to visually inspect if the input is correct.

I created this sheet, that uses the data from the main one and formatted the exclusion statements using new lines for simplicity: https://docs.google.com/spreadsheets/d/1mhPPdmVyGTZHfhnCTK9pkyuVCwIFm5JtKiTa011e9g8/edit#gid=1755020585

Reply:
thank you, based on discussions in DM we'll use the whole data, i agree we should strip everything from the original exclusions column and just leave one on each line like that, the parsing is incovenient and the CSV blob is very hard to read